repo2docker/Project.toml

```diff
@@ -0,0 +1,9 @@
+[deps]
```

Comment:
These aren't private keys or anything are they?

Reply:
Within Julia, those keys are [basically the version number](https://julialang.github.io/Pkg.jl/v1/toml-files/#The-uuid-field-1). Nothing private to it.